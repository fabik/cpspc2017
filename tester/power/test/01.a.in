100
572 554
377 953
693 171
765 223
349 271
389 297
276 980
694 425
895 348
740 983
846 808
790 650
391 985
351 166
377 570
183 80
895 144
910 276
871 379
884 293
33 492
680 856
760 826
567 345
340 416
340 453
5 405
205 935
884 905
119 970
66 696
384 231
517 510
325 885
307 822
404 734
308 237
442 227
713 415
294 702
221 434
239 512
587 792
119 496
944 260
460 533
556 258
194 136
621 480
721 787
506 760
615 67
75 825
162 14
280 328
532 747
985 341
551 812
753 378
51 801
725 366
510 767
863 918
989 36
397 714
526 124
81 814
505 475
867 544
659 151
351 573
148 982
915 894
868 978
48 781
122 286
70 675
12 682
735 912
317 137
542 966
835 641
442 417
77 956
596 3
690 159
859 778
822 603
492 115
857 312
476 791
591 965
419 334
324 310
942 911
977 504
851 302
518 50
985 360
868 392
