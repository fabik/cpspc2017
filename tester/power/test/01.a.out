884
87
1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 57 58 59 60 61 64 66 68 69 70 72 73 74 76 78 79 80 81 82 83 84 85 87 88 89 90 91 92 93 94 95 96 98
13
39 56 62 63 65 67 71 75 77 86 97 99 100
